app/views/videos/show/_translate_frame.html.erb

```diff
@@ -31,18 +31,21 @@
             <% end %>
           </div>
           <div role="tabpanel" class="tab-pane fade tab-srt" id="tab-srt">
+            <span>If you want to translate offline using an SRT, you can
             <% if (srt_link = video.amara_en_srt) %>
-              <span>If you want to translate offline using an SRT, you can <b><%= link_to 'download the video\'s SRT', srt_link %></b>. When you're finished translating, just ship it back to us!</span>
+              <b><%= link_to 'download the video\'s SRT', srt_link %></b>.
             <% else %>
-              <span>If you want to translate offline using an SRT, you can use the video above. Unfortunately, there is no English SRT available. When you're finished translating, just ship it back to us!</span>
+              use the video above. Unfortunately, there is no English SRT available.
             <% end %>
+            When you're finished translating, use the following steps to get credit for your work:</span>
 
-            <%= form_tag(video_translation_upload_path(video, translation), :class => 'tab-form', :multipart => true) do %>
-              <div class="form-group">
-                <%= file_field_tag 'srt', :class => 'filestyle', :'data-icon' => false %>
-              </div>
-              <%= submit_tag 'Upload', :class => 'btn btn-primary' %>
-            <% end %>
+            <ol class="upload-steps">
+              <li>Visit the <b><%= link_to 'Amara link', video.amara_link, :target => '_amara' %></b>.</li>
+              <li>Hover over the wrench and click <kbd>Upload Subtitles</kbd>.</li>
+              <li>Follow the steps to finish subtitling the video, and click <kbd>Publish</kbd> to complete.</li>
+              <li>In the languages panel, click <kbd>Burmese</kbd>. Copy the URL of the page.</li>
+              <li>Navigate to the tab on the left, "Translate on Amara," on this page. Paste your link and hit <kbd>Submit</kbd>.</li>
```

Comment:
Just a bit strange to have it in quotes. Maybe just bold it to match the tab panel.